Dockerfile-linux.template

```diff
@@ -176,7 +176,7 @@ RUN set -eux; \
 		--with-lto \
 {{ ) end -}}
 		--with-system-expat \
-		--without-ensurepip \
+		--with-ensurepip \
```

Comment:
Btw `--with-ensurepip` is the default, so we could also omit this configure option (I left it in to be explicit, but equally happy to remove it).

Reply:
Yeah, I'm a fan of being explicit about this -- I wish we could somehow pass in `--default-pip` from here (https://docs.python.org/3/library/ensurepip.html#command-line-interface), but I don't think there's a way. :smile:

Reply:
It looks like the only way would be to switch from `make install` to `make altinstall`:
https://github.com/python/cpython/blob/v3.12.5/Makefile.pre.in#L1903-L1922

...though I haven't dug into the full implications.

Reply:
Yeah, I think we're good -- making a symlink isn't too bad, especially when we're already making a bunch. :smile:

Reply:
> It looks like the only way would be to switch from `make install` to `make altinstall`:

And in fact I'm getting confused between `--altinstall` vs `--default-pip` vs the defaults (I don't find the usage text for ensurepip very clear), it's `--default-pip` we need, and there is no make target for that anyway.